xbmc/video/dialogs/GUIDialogTeletext.cpp

```diff
@@ -79,6 +79,29 @@ bool CGUIDialogTeletext::OnMessage(CGUIMessage& message)
   return CGUIDialog::OnMessage(message);
 }
 
+void CGUIDialogTeletext::DoProcess(unsigned int currentTime, CDirtyRegionList &dirtyregions)
+{
+  if (m_pTxtTexture)
+  {
+    m_TextDecoder.RenderPage();
```

Comment:
this is wrong, we don't render in DoProcess. why did you pull this to this place?